common.yml

```diff
@@ -88,14 +88,15 @@
   post_tasks:
     - name: Register node.
       shell: >
-        tsuru-admin docker-node-add --register address=http://{{ gce_private_ip }}:{{ docker_port }}
+        tsuru-admin docker-node-add --register address=http://{{ gce_private_ip }}:{{ docker_port }} pool=default
```

Comment:
Will this need to be applied manually to the CI environment with `docker-node-update`?